src/Operations/ConvertContoursToMeshes.cc

```diff
@@ -476,11 +501,34 @@ bool ConvertContoursToMeshes(Drover &DICOM_data,
                 return;
             };
 
+            // removes contours in base_cops which are identical to contours in cops_to_remove
+            // returns list of contours remaining in base_cops, but does not modify base_cops
+            // this is used instead of base_cops.erase(...) since erasing an object with a reference wrapper will erase all references to it,
+            // causing seg faults if other list of contours store the same contour
+            // https://stackoverflow.com/a/2874533
```

Comment:
Wait so is this the stl container problem or the set iterator problem described in the stackoverflow post?

Reply:
outdated, was an stl container problem